src/device/sam/atsamd51x-bitfields.go

```diff
@@ -0,0 +1,63 @@
+// Hand created file. DO NOT DELETE.
+// atsamd51x bitfield definitions that are not auto-generated by gen-device-svd.go
+
+// +build sam,atsamd51
+
+// These are the supported pchctrl function numberings on the atsamd51x
+// See http://ww1.microchip.com/downloads/en/DeviceDoc/SAM_D5xE5x_Family_Data_Sheet_DS60001507F.pdf
+// table 14-9
+
+package sam
+
+const (
+	PCHCTRL_GCLK_OSCCTRL_DFLL48     = 0 // DFLL48 input clock source
+	PCHCTRL_GCLK_OSCCTRL_FDPLL0     = 1 // Reference clock for FDPLL0
+	PCHCTRL_GCLK_OSCCTRL_FDPLL1     = 2 // Reference clock for FDPLL1
+	PCHCTRL_GCLK_OSCCTRL_FDPLL0_32K = 3
+	PCHCTRL_GCLK_OSCCTRL_FDPLL1_32K = 3
+	PCHCTRL_GCLK_SDHC0_SLOW         = 3
+	PCHCTRL_GCLK_SDHC1_SLOW         = 3
+	// GCLK_SERCOM[0..7]_SLOW = 3
+	// FDPLL0 = 3 // 32KHz clock for internal lock timer
+	// FDPLL1 = 3 //32KHz clock for internal lock timer
+	// SDHC0 = 3 // Slow
+	// SDHC1 = 3 // Slow
+	PCHCTRL_GCLK_EIC          = 4
+	PCHCTRL_GCLK_FREQM_MSR    = 5 // FREQM Measure
+	PCHCTRL_GCLK_FREQM_REF    = 6 // FREQM Reference
+	PCHCTRL_GCLK_SERCOM0_CORE = 7 // SERCOM0 Core
+	PCHCTRL_GCLK_SERCOM1_CORE = 8 // SERCOM1 Core
+	PCHCTRL_GCLK_TC0          = 9
+	PCHCTRL_GCLK_TC1          = 9  // TC0, TC1
+	PCHCTRL_GCLK_USB          = 10 // USB
+	//22:11 GCLK_EVSYS[0..11] EVSYS[0..11]
```

Comment:
I think we can define it.

PCHCTRL_GCLK_EVSYS0
..
PCHCTRL_GCLK_EVSYS11

Reply:
I went ahead and added all the rest of the bitfields for the PCHCTRLm mapping.